packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/java/org/eclipse/sirius/components/collaborative/diagrams/dto/DiagramRefreshedEventPayload.java

```diff
@@ -29,6 +29,8 @@ public record DiagramRefreshedEventPayload(UUID id, Diagram diagram, String caus
 
     public static final String CAUSE_LAYOUT = "layout";
 
+    public static final String CAUSE_RELOAD = "reload";
```

Comment:
There's a bug here: the new value is not added in the GraphQL Schema and not handled by the front (in `DiagramRenderer.tsx`). I'm investigating how exactly it should be handled.